### PR TITLE
docs: Translate some docs

### PR DESCRIPTION
**Pull request by youngjae99**
## Overview

Migrating and translating some jsdoc to i18n documentations

## PR Checklist

- [x] I read and included theses actions below

1. I have read the [Contributing Guide](https://github.com/toss/slash/blob/main/.github/CONTRIBUTING.md)
2. I have written documents and tests, if needed.

### CODE REVIEW

File: packages/common/hangul/src/utils.en.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: utils
+---
+
+# hasBatchim
+
+Check if a hangul string has a [final consonants (batchim)](https://en.wikipedia.org/wiki/Hangul_consonant_and_vowel_tables) at the end.
```
Comment (HoseungJang) [inline packages/common/hangul/src/utils.en.md:7]: `hangul/src/utils.ts` is not exported from `@toss/hangul` so I think it doesn't have to be added to our documentation.

**Review by HoseungJang — CHANGES_REQUESTED** [review]

File: packages/common/utility-types/src/serializable.ts [hunk 2]
```diff
@@ -1,11 +1,5 @@
 type Primitive = undefined | null | boolean | number | symbol | string;
```
[the remark this answers is not in the document]
Comment (youngjae99) [reply packages/common/utility-types/src/serializable.ts:2]: > If you don't use jsdoc, I think you'd better add this comment! at top of this file📌 `/** @tossdocs-ignore */`

My mistake. Thanks for checking!

File: packages/common/utils/src/reverseKeyValue.en.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# reverseKeyValue
+
+Returns an object in the form of { [value]: key } by replacing key and value
```
Comment (raon0211) [inline packages/common/utils/src/reverseKeyValue.en.md:3]:
```suggestion
Returns an object in the form of { [value]: key } by swapping key and value
```

**Review by raon0211 — APPROVED** [review]
Thanks, @youngjae99 !

**Pull request merged**